novice-goals.Rmd

```diff
@@ -102,6 +102,46 @@ but nothing more.
 
 -   How can I represent and manage missing values?
     -   NA
+    Python Learning Objectives
+    -   Identify missing values (e.g., NaN) in your datasets
+    -   Examine the impact of missing valus on your datasets
+    -   Demonstrate and use commands that process missing values
+    -   Demonstrate how to replace missing values
+
+    Python Lecture Notes (general info) 
+    -   Missing Completely at Random (MCAR)
+    -   Missing at Random (MAR)
+    -   Missing Not at Random (MNAR)
+    -   Missing data at scale (what's a little and what's alot?)
+    -   Imputation and Multiple Imputation 
+    -   When summing the data, missing values will be treated as zero
+    -   If all values are missing, the sum will be equal to NaN
+    -   cumsum() and cumprod() methods ignore missing values but preserve them in the resulting arrays
+    -   Missing values in GroupBy method are excluded (just like in R)
+    -   Many descriptive statistics methods have skipna option to control if missing data should be excluded . This value is set to True by default (unlike R) 
+
+    Python Lecture Notes (useful commands)
+    -   Table: df.method() / description
+    -   dropna() / drop missing observations
+    -   dropna(how='all') / drop observations where all cells are NA
+    -   dropna(axis=1,how='all') / drop coloumn if all values are missing
+    -   dropna(thresh=5) / drop rows that contain less than 5 non-missing values
+    -   fillna(0) / replace missing values with zeros
+    -   isnull() returns True if the value is missing
+    -   isnotnull() / Returns True for non-missing values
```

Comment:
This is without the "is"-part, and similarly to above, I suggest we use `notna()` instead.